src/cache/asset.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'file_path'
+require 'data'
+
+module Metalware
+  module Cache
+    class Asset
+      def data
+        @assets ||= begin
+          Data.load(FilePath.asset_cache).tap do |a|
+            a.merge! blank_cache if a.empty?
```

Comment:
Try not using `merge!` if it isn't required, it can have some nasty side affects if you aren't careful. I am aware a lot of existing code is implemented in this manor, however this should also be changed at some point (ideally).

In this case, something along these lines will work:
```
raw_load = Data.load(path)
raw_load.empty? ? blank_cache : raw_load
```